File: src/main/java/uk/yermak/audiobookconverter/ArtWork.java
package uk.yermak.audiobookconverter;

public interface ArtWork {
    String[] IMAGE_EXTENSIONS = new String[]{"jpg", "jpeg", "jfif", "png", "bmp"};

    long getCrc32();

    String getFileName();

    boolean matchCrc32(long crc32);
}

        

File: src/main/scala/uk/yermak/audiobookconverter/ArtWork.scala
package uk.yermak.audiobookconverter

/**
  * Created by yermak on 1/11/2018.
  */
trait ArtWork {
  def getCrc32: Long

  def setCrc32(crc32: Long): Unit

  def getFileName: String

  def setFileName(fileName: String): Unit

  def getFormat: String

  def setFormat(format: String): Unit

}
